GiEnJul/Features/GenericRepository.cs

```diff
@@ -64,6 +67,28 @@ public async Task<T> DeleteAsync(string partitionKey, string rowKey)
             return await DeleteAsync(entity);
         }
 
+        public async Task<List<T>> DeleteAsyncMultiple(List<T> entities)
+        {
+            List<T> deletedEntities = new List<T>();
+            foreach (var addedPerson in entities)
+            {
+                try
+                {
+                    var result = await DeleteAsync(addedPerson);
+                    deletedEntities.Add(result);
+                }
+                catch (Exception) { }
```

Comment:
Log or keep these exceptions in a list to deal with later?
Currently we don't know what caused the delete to fail.

Reply:
Fixed this by using batches.